.flake8

```diff
@@ -70,6 +70,7 @@ ignore =
     E713,
     E712,
     B028,
+    W604,
```

Comment:
Added this because pre-commit would block otherwise and replacing all backticks with `repr()` is out of scope here.